src/interface/news/RegularArticle.tsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { ReactNode } from 'react';
 import { Link } from 'react-router-dom';
 
 import Contributor from 'interface/ContributorButton';
 import makeNewsUrl from 'interface/news/makeUrl';
 import Panel from 'interface/others/Panel';
+import { ContributorType } from 'interface/ContributorButton/ContributorButton';
 
-const RegularArticle = props => {
+interface Props {
+  title: ReactNode;
+  bodyStyle?: object;
```

Comment:
This should be `React.CSSProperties`

Reply:
I used object because the Panel component used object as a type for bodystyle prop

Reply:
That is something that can be cleaned up as part of this PR. CSS props should have the correct type.

Reply:
I've made the changes